src/sprites/sprite.js

```diff
@@ -140,11 +149,41 @@ class Sprite {
         const blocksContainer = this.blocks._blocks;
         const originalBlocks = Object.keys(blocksContainer).map(key => blocksContainer[key]);
         const copiedBlocks = JSON.parse(JSON.stringify(originalBlocks));
-        newBlockIds(copiedBlocks);
+        const oldToNew = newBlockIds(copiedBlocks);
         copiedBlocks.forEach(block => {
             newSprite.blocks.createBlock(block);
         });
 
+        Object.keys(this.comments).forEach(commentId => {
+            const comment = this.comments[commentId];
+            const newComment = new Comment(
+                null, // use new comment id
+                comment.text,
+                comment.x,
+                comment.y,
+                comment.width,
+                comment.height,
+                comment.minimized
+            );
+            if (comment.blockId) {
+                const newBlockId = oldToNew[comment.blockId];
+                if (newBlockId) {
+                    newComment.blockId = newBlockId;
+                    const newBlock = newSprite.blocks.getBlock(newBlockId);
+                    if (newBlock) {
+                        newBlock.comment = newComment.id;
+                    } else {
+                        log.warn(`Could not find block with id ${newBlockId
+                        } associated with comment ${newComment.id}`);
```

Comment:
```suggestion
                        log.warn(`Could not find block with id ${newBlockId} associated with comment ${newComment.id}`);
```
Exactly 120 characters long, so it barely passes lint